src/libs/auth.js

```diff
@@ -197,6 +198,14 @@ authStore.subscribe(async (state, oldState) => {
   }
 })
 
+authStore.subscribe(async (state, oldState) => {
+  if (oldState.registrationStatus === 'unregistered' && state.registrationStatus === 'registered') {
```

Comment:
What do you think about going ahead with that change vs merging as it is now, or maybe leaving a comment or TODO there to indicate a possible future improvement?

Reply:
I think a comment would be good, perhaps in `PageViewReporter` since that's what ultimately gets affected.

Reply:
I just chatted with Amelia and the main question they are looking to get at is the conversion rate from people who browse anonymously and leave vs people who then register to use the site. So I think it might be worthwhile to try @panentheos 's suggestion in this PR so we don't muddy the data with orphaned anon events waiting to add it later